src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1362,7 +1363,7 @@ void window_guest_stats_paint(rct_window *w, rct_drawpixelinfo *dpi)
     y += 10;
     gfx_draw_string_left(dpi, STR_GUEST_STAT_ENERGY_LABEL, gCommonFormatArgs, COLOUR_BLACK, x, y);
 
-    sint32 energy = ((peep->energy - 32) * 85) / 32;
+    sint32 energy = ((peep->energy - PEEP_MIN_ENERGY) * 255) / (PEEP_MAX_ENERGY - PEEP_MIN_ENERGY);
```

Comment:
Is this correct? You've changed the numbers but not explained why

Reply:
He did. See the opening post.

255 / 96 === 85 / 32.